### DISCUSSION TITLE
How to verify image release version

### DISCUSSION THREAD

**Discussion by robertreems** [actions/virtual-environments#7198] [General questions]
Hi,

My customer builds the images based on the latest tag of ubuntu22/* and ubuntu20/*
We'd like to verify the version number/release of the image when the image is being used on a pipeline.

I'm not 100% certain that we do use the stable versions since the Azure DevOps 'initialize job' shows: `current image version: 'dev' `

We've found the file /imagegeneration/imagedata.json but this also suggests 'dev'.

So are we using non production ripe versions of the image (currently ubuntu22/20230219.1)? And secondly is there a method in viewing the release version from a running agent?

**Comment by panekj** [comment]
This is a variable that you're supposed to set yourself when building a specific release image.
Since this repository is constantly moving, it is not proper to set a specific tag in code.

**Reply by robertreems** [reply]
ok, thanks for the quick anser!

And what about the current image variable that always shows 'dev'. Is this normal and am I correct to assume the image is 'stable' for production use when I build an image from a checked out release?
